packages/ckeditor5-dev-utils/lib/compiler/resolveimportpath.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * @license Copyright (c) 2003-2016, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const path = require( 'path' );
+const fs = require( 'fs' );
+const getWorkspaceRelativePathInfo = require( './getworkspacerelativepathinfo' );
+
+/**
+ * Looks for the imported path in context of the requester path and returns informations about it when the request matches some file.
+ *
+ * @param {String} requesterPath Path to the requester.
+ * @param {String} importPath Path to the imported file.
+ * @param {Array.<String>} packagePaths Array of node_modules directories and packages where packages will be looking for.
+ * @returns {Object|null}
+ */
+module.exports = function resolveImportPath( requesterPath, importPath, packagePaths ) {
+	const requestPackageName = getWorkspaceRelativePathInfo( importPath ).packageName;
```

Comment:
This code makes little sense when you read it – import path can't be "workspace relative". Besides, it would be `requestedPackageName`.

So, if anything, there could be a generic function called `parseImportPath()`.